datafusion/core/src/datasource/file_format/parquet.rs

```diff
@@ -75,6 +75,17 @@ use crate::physical_plan::{
     Statistics,
 };
 
+/// Size of the buffer for [`AsyncArrowWriter`].
+const PARQUET_WRITER_BUFFER_SIZE: usize = 10485760;
```

Comment:
I don't know why these values are (slightly) different, but I figured we could start by keeping them the same and then could unify them as a follow on if needed